core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -3128,7 +3128,7 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
     * @tparam T
     * @return
     */
-  private def withBillingProjectContext[T](billingProjectName: String, parentContext: RawlsRequestContext)(
+  def withBillingProjectContext[T](billingProjectName: String, parentContext: RawlsRequestContext)(
```

Comment:
Exposed this method and modified it to work with both GCP and Azure billing projects.